src/sai/Codingexam1/ReturnIndices.java

```diff
@@ -0,0 +1,21 @@
+package sai.Codingexam1;
+
+public class ReturnIndices {
+
+	public static void main(String[] args) {
+		ReturnIndices returnIndices = new ReturnIndices();
+		int[] arr = { 2, 11, 15, 7, 4 };
+		returnIndices.addNumber(arr);
+	}
+
+	void addNumber(int[] arr) {
+		for (int index = 0; index < arr.length; index++) {
+			for (int i = 1; i < arr.length; i++) {
```

Comment:
Inner for loop should from i+1, think on this, if you want to connect on Skype then feel free to message.

Take reference of this code
Kangan - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/987/files
Rupali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/991/files
Dipali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1003/files